deploy/generate_addon_csi.sh

```diff
@@ -176,7 +176,7 @@ generate_cloud_config() {
   - encoding: b64
     content: ${KUBECONFIG_B64}
     owner: root:root
-    path: /etc/kubernetes/cloud-config
+    path: /var/lib/rancher/rke2/etc/config-files/cloud-provider-config
```

Comment:
IIRC, the path of `/etc/kubernetes/cloud-config` is default to rke1 or k3s? and the new path is valid for rke2 only? can we check if by the input provide type.

Reply:
This path depends on harvester-csi-driver.
For RKE1, the addon yaml includes the related information.
For rke2/k3s, this is the same because of the harvester-csi-driver definition.

Reply:
got it, thanks.